packages/integration-aws-node/api-report.md

```diff
@@ -0,0 +1,77 @@
+## API Report File for "@backstage/integration-aws-node"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { AwsCredentialIdentityProvider } from '@aws-sdk/types';
+import { Config } from '@backstage/config';
+
+// @public
+export type AwsCredentialProvider = {
+  accountId?: string;
+  stsRegion?: string;
+  sdkCredentialProvider: AwsCredentialIdentityProvider;
+};
+
+// @public
+export type AwsCredentialProviderOptions = {
+  accountId?: string;
+  arn?: string;
+};
+
+// @public
+export interface AwsCredentialsManager {
+  getCredentialProvider(
+    opts?: AwsCredentialProviderOptions,
+  ): Promise<AwsCredentialProvider>;
+}
+
+// @public
+export type AwsIntegrationAccountConfig = {
+  accountId: string;
+  accessKeyId?: string;
+  secretAccessKey?: string;
+  profile?: string;
+  roleName?: string;
+  partition?: string;
+  region?: string;
+  externalId?: string;
+};
+
+// @public
+export type AwsIntegrationConfig = {
+  accounts: AwsIntegrationAccountConfig[];
+  accountDefaults: AwsIntegrationDefaultAccountConfig;
+  mainAccount: AwsIntegrationMainAccountConfig;
+};
+
+// @public
+export type AwsIntegrationDefaultAccountConfig = {
+  roleName?: string;
+  partition?: string;
+  region?: string;
+  externalId?: string;
+};
+
+// @public
+export type AwsIntegrationMainAccountConfig = {
+  accessKeyId?: string;
+  secretAccessKey?: string;
+  profile?: string;
+  region?: string;
+};
+
+// @public
+export class DefaultAwsCredentialsManager implements AwsCredentialsManager {
+  // (undocumented)
+  static fromConfig(config: Config): DefaultAwsCredentialsManager;
+  getCredentialProvider(
+    opts?: AwsCredentialProviderOptions,
+  ): Promise<AwsCredentialProvider>;
+}
+
+// @public
+export function readAwsIntegrationConfig(config: Config): AwsIntegrationConfig;
```

Comment:
Looks like this is only used in internal tests currently. I suggest that we skip exporting this and the related types until we need them in the future. That should greatly reduce the API surface.